### PR TITLE
Add gateway downlink info in TxAck message

### PR DESCRIPTION
**Pull request by sagar-patel-sls**
This exposes the tx_info field, holding all the TX parameters
to TxAck Event

- fix flush_profiles_cache_test.go errors.

Fixes [brocaar/chirpstack-application-server#537](https://github.com/brocaar/chirpstack-application-server/issues/537)

Signed-off-by: SAGAR PATEL <sagar.a.patel@slscorp.com>

### CODE REVIEW

File: internal/downlink/ack/ack_test.go [hunk 1]
```diff
@@ -121,6 +121,7 @@ func TestDownlinkAck(t *testing.T) {
 			HandleTxAckRequest: &as.HandleTxAckRequest{
 				DevEui: []byte{1, 2, 3, 4, 5, 6, 7, 8},
 				FCnt:   10,
+				TxInfo: &txInfo1,
```
Comment (brocaar) [inline internal/downlink/ack/ack_test.go:124]: Isn't the `GatewayId` expected in `HandleTxAckRequest`?

Comment (sagar-patel-sls) [reply internal/downlink/ack/ack_test.go]: Added `GatewayId` ub test

Thanks

**Comment by hoffi085** [comment]
I updated the network-server to 3.12.3, but nothing changed in the txack message:
image

**Comment by sagar-patel-sls** [comment]
@hoffi085 
You need to update the `chirpstack-application-server` for `txack` message, so wait for the next release of the `chirpstack-application-server`.

Thanks

**Comment by hoffi085** [comment]
I see, thanks!

**Pull request merged**